### PR TITLE
entry: new cask

### PR DESCRIPTION
**Pull request by grabic1060**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/entry.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  uninstall pkgutil: [
+    "org.playentry.entry.*",
+    "org.playentry.entry.pkg",
+  ]
+
+  zap trash: [
+    "~/Library/Application Support/Entry",
+    "~/Library/Application Support/entry-hw",
+    "~/Library/Caches/io.sentry",
```
Comment (bevanjkay) [inline Casks/entry.rb:25]: @grabic1060 This line looks like it's for a different app, it was matched by `brew createzap` because it includes "entry".

**Review by bevanjkay — DISMISSED** [review]

**Comment by bevanjkay** [comment]
@grabic1060 One of our maintainers should be able to help with livecheck in the next day or two

**Comment by grabic1060** [comment]
@bevanjkay I looked those path, found it line 23 and 24 are correct but line 25 is for a different app. Would I have to fetch upstream my fork before I fix it?

**Comment by razvanazamfirei** [comment]
Hi, @grabic1060 ! Thanks so much for your first contribution! This looks great! 👍

**Review by razvanazamfirei — APPROVED** [review]

**Pull request merged**